redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
 Please refer to the <<ReleaseNotesRedfish>> section for further information
 about the power management actions and statuses.
 
+=== Physical Server Provisioning
+{redfish-provider-full} lets the users request to initiate network booting of
+an Operating System.

```

Comment:
It feels like we only tell first half of the story. I would reword it a little to emphasize that we allow for OS installment, not just network boot. Something like:

_{redfish-provider-full} allows user to install operating system on physical server by leveraging PXE network boot mechanism. Basically, what happens is that physical server is rebooted with "boot from LAN" option set while PXE server is connected to the same LAN and pre-configured to install selected operating system on that specific physical server. Steps below demonstrate how to initialize provisioning for one or more physical servers._

Reply:
👍 
I worded it differently, but in the suggested spirit.